website/docs/r/virtual_desktop_application_group.html.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-subcategory: "Desktop Virtualization"
+subcategory: "DesktopVirtualization"
```

Comment:
I don't think we should be changing this?

Reply:
Yeah, I got called out by the linter so I changed it to what was in the registration code... I'll switch it back to what it was originally. 😄